app/models/classification.rb

```diff
@@ -88,7 +86,7 @@ def published_detailed_guides
   end
 
   def published_policies
```

Comment:
Assuming those `@policy` instance variables in the frontend controllers are no longer required, I think this method can be removed completely.